### ISSUE TITLE
Brightness/Warmth changes don't follow quick tapping gesture

### ISSUE THREAD

**Issue by pblasi** [koreader/koreader#6569]
* KOReader version: 2020.08.01
* Device: Kobo Clara HD

First tap makes the increment/decrease and prompt message "Frontlight intensity set to whatever" appears. Then it seems next quick tap is used to dismiss the previous prompt message (as it's still showing) but no new increment/descrease is applied. Any easy way to disable the prompt message to test?

Thanks!

**Comment by Frenzie** [comment]
Didn't check, but you're probably looking for what happens when you comment out these lines:

https://github.com/koreader/koreader/blob/b5d33058761625111d176123121bcc881864a64e/frontend/device/devicelistener.lua#L50-L53

**Comment by pblasi** [comment]
Exactly! Removing these lines fixes the issue. Now brightness/warmth changes follow the quick tapping. Also I get rid of the message which is unnecessary to me since I have brightness level in the status bar. 

1. If you want to keep the messages, would be possible making them to ignore taps which dismiss them?

2. Option in the UI to disable these messages?

Thanks!
